spec/services/import_matches_spec.rb

```diff
@@ -0,0 +1,199 @@
+      allow(GoalComWrapper).to receive(:get_round).with(2) { result_from(matches_table_page_2) }
+
+      expect do
+        ImportMatches.call
+      end.to change{ FootballMatch.count}.by(4).and change{ Round.count }.by(2)
+
+       # match_array
+      expect(Round.all).to include(
+        have_attributes(year: 2017, number: 1),
+        have_attributes(year: 2017, number: 2),
+      )
+
+      round_first = Round.find_by(year: 2017, number: 1)
+      round_second = Round.find_by(year: 2017, number: 2)
+
+      expect(FootballMatch.all).to include(
+        have_attributes(match_date: DateTime.parse("2017/09/10 12:30"), home_team_id: monaco.id, away_team_id: porto.id, home_team_score: nil, away_team_score: nil, round_id:round_first.id ),
+        have_attributes(match_date: DateTime.parse("2017/09/11 12:45"), home_team_id: real.id, away_team_id: barcelona.id, home_team_score: nil, away_team_score: nil, round_id: round_first.id),
+        have_attributes(match_date: DateTime.parse("2017/11/20 12:30"), home_team_id: monaco.id, away_team_id: real.id, home_team_score: nil, away_team_score: nil, round_id: round_second.id),
+        have_attributes(match_date: DateTime.parse("2017/11/21 12:45"), home_team_id: barcelona.id, away_team_id: porto.id, home_team_score: nil, away_team_score: nil, round_id: round_second.id),
+      )
+
+      matches_table_page_1_after_call = <<~RATE_PLAN
+        WEEK 1
+        Time              | Team  home  | Team away     | Score away | Score home | completed
+        ------------------+-------------+---------------+------------+------------------------
+        2017/09/10 12:30  | Monaco      |   Porto       |     1      |      5     | true
+        2017/09/11 12:45  | Real Madryt |   Barcelona   |     0      |      0     | false
+       RATE_PLAN
+      matches_table_page_2_after_call = <<~RATE_PLAN
+      WEEK 2
+        Time              | Team home   | Team away     | Score away | Score home | completed
+        ------------------+-------------+---------------+-------------------------------------
+        2017/11/20 12:30  | Monaco      |   Real Madryt |      0     |      0     | false
+        2017/11/21 12:45  | Barcelona   |   Porto       |      0     |      0     | false
+      RATE_PLAN
+
+
+      allow(GoalComWrapper).to receive(:get_round).with(1) { result_from(matches_table_page_1_after_call) }
+      allow(GoalComWrapper).to receive(:get_round).with(2) { result_from(matches_table_page_2_after_call) }
+
+      expect do
+        ImportMatches.call
+      end.to not_change{ FootballMatch.count}.and not_change{ Round.count }
+
+      round_first = Round.find_by(year: 2017, number: 1)
+      round_second = Round.find_by(year: 2017, number: 2)
+
+      expect(FootballMatch.all).to include(
+        have_attributes(match_date: DateTime.parse("2017/09/10 12:30"), home_team_id: monaco.id, away_team_id: porto.id, home_team_score: 1, away_team_score: 5, round_id:round_first.id ),
```

Comment:
`DateTime.parse("2017/09/10 12:30")` -> `DateTime.new(2017, 9, 10, 12, 30)`

Reply:
@stevo 
we chose parse DateTime.parse("2017/09/10 12:30") because is represent  this in our Tabele. Example 
2017/09/10 12:30  | Monaco      |   Porto       |     0      | but of course we can use new